backend/src/posts/posts.controller.ts

```diff
@@ -12,9 +12,12 @@ export class PostsController {
   constructor(private readonly postsService: PostsService) { }
 
   @ApiBearerAuth()
-  @Personal() //provides @UserObjectID to get userid
+  // @Personal() //provides @UserObjectID to get userid
```

Comment:
This is required for authentication. Otherwise anyone can create a post as anyone else